locale/templates/settings.ftl

```diff
@@ -214,6 +216,10 @@ mvs-enter-verification-code-desc = Please enter the verification code that was s
 msv-cancel-button = Cancel
 msv-submit-button = Verify
 
+## Auth-server based errors that originate from backend service
```

Comment:
Is  this comment for a single string or a section?

Reply:
Hi @peiying2 this string should have been in the same section as the other `auth-error-xxx` strings. 

This seems to be an issue in the code base, I'll make the necessary updates so that this duplicate section is removed and the string is is added to the other `auth-error-xxx` strings.

Reply:
Thanks for the fix